source/auth/auth.rst

```diff
@@ -1210,24 +1210,12 @@ username.
     description: "Server's reply to clientStep1"
     strict: false
     fields:
```

Comment:
Just a note – we know it’s likely that in the future, other information will need to be passed from the driver to callbacks, e.g. the SRV hostname for resource indicators in the sense of RFC8707. Does it make sense to prepare for that? For example, instead of passing `principalName` as a string parameter to the callbacks, it could be a BSON document like `OIDCMechanismServerStep1` currently is.

Reply:
I merged the `timeout` parameter into that object as well